bot/commands.js

```diff
@@ -0,0 +1,236 @@
+// Commands
+const commands = require('./commands/commands.js');
+const games = require('./commands/games.js');
+const help = require('./commands/help.js');
+const my100status = require('./commands/my100status.js');
+const playingnow = require('./commands/playingnow.js');
+const popcorn = require('./commands/popcorn.js');
+const stats = require('./commands/stats.js');
+const test = require('./commands/test.js');
+
+// Command Prefixes
+const adminPrefix = '-';
+const userPrefix = '$';
+
+const testbotjoe26Tag = "testbotjoe26#8213";
+
+var execute = function (bot, msg, command, parameters)
+{
+    switch (true)
+    {
+        case (command === userPrefix + 'commands'): // $commands
+
+            commands(msg);
+
+            break;
+
+        case (command === userPrefix + 'games'): // $games
+
+            games(msg);
+
+            break;
+
+        case (command === userPrefix + 'help'): // $help
+
+            parameters[0] = parameters[0].replace(/\$/g, "");
```

Comment:
Found a bug while testing.

When issuing the $help command with no parameters, the parameters variable is undefined and thus caused an error to occur when the replace function was called.

Reply:
This has been fixed.